client/src/pages/Login.jsx

```diff
@@ -87,24 +87,16 @@ function Login(props: Props) {
         </Form.Item>
         <div className="red-text">{error}</div>
         <Form.Item>
-          {getFieldDecorator('remember', {
-            valuePropName: 'checked',
-            initialValue: true,
-          })(
-            <Checkbox className="form-checkbox">
-              <div className="white-text">Remember me</div>
-            </Checkbox>,
-          )}
-          <Link className="form-forgot" to="/password-reset">
-            Forgot password
-          </Link>
           <Button type="primary" htmlType="submit" className="form-button">
             Log In
           </Button>
           <div className="white-text">Don&#39;t have an account?</div>{' '}
           <Link className="link-now" to="/register">
             Register Now!
           </Link>
+          <Link className="form-forgot" to="/password-reset">
+            Forgot password
```

Comment:
Should there be a "?" here? I feel like usually there is (also capitalize "password" for consistency)

Reply:
I like the "?", but just looking on other random sites it seems they don't capitalize